### PR TITLE
[Table Designer] Add "New Table" menu to Tables node in OE

### PR DESCRIPTION
**Pull request by caohai**
This PR adds "New Table" context menu to Tables node in OE.
Screen Shot 2022-01-10 at 01 15 02

The end-to-end workflow to add a new table and persist the table to db works fine. (Along with my changes to DacFx)
image
image

### CODE REVIEW

**Comment by alanrenmsft** [comment]
does it depend on DacFx changes that is not in current STS version? @caohai

**Comment by caohai** [comment]
> does it depend on DacFx changes that is not in current STS version? @caohai

The button itself can show up with current STS version but it would fail to open the designer without related DacFx change. 

Do you think it's ok to add the button this way? Or do you think we should do this **after** merging the DacFx changes and bumping STS version?

**Comment by alanrenmsft** [comment]
@caohai I prefer to do this after the STS vbump.

**Comment by caohai** [comment]
> @caohai I prefer to do this after the STS vbump.

Sure I'll ping you again for this PR once we are done with the DacFx change and vbump.

File: extensions/mssql/package.json [hunk 1]
```diff
@@ -503,6 +503,11 @@
           "command": "mssql.designTable",
           "when": "connectionProvider == MSSQL && nodeType == Table && config.tableDesigner.enableFeature",
           "group": "0_query@3"
+        },
+        {
+          "command": "mssql.newTable",
+          "when": "connectionProvider == MSSQL && nodeType == Folder && nodeLabel == Tables && config.tableDesigner.enableFeature",
```
Comment (chlafreniere) [inline extensions/mssql/package.json:509]: what's the difference between the "Folder" and "Table" nodeType (5 lines above)?

Comment (chlafreniere) [reply extensions/mssql/package.json:509]: Oh, this is the "Tables" folder. Read too fast 😄

**Comment by caohai** [comment]
@alanrenmsft

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**